### ISSUE TITLE
Response time

### ISSUE THREAD

**Issue by Monter** [digibart/upscuits#23]
Is there a plan to display the response time of the monitored object? API can return such data by set "responseTimes" and "responseTimesAverage" in getMonitors.

**Comment by digibart** [comment]
nope, maybe the `responseTimesAverage`, but showing all `responseTimes` would get too complicated I think. Not only the implementation but also making sure the interface stays nice and clean....
